### PR TITLE
UI edits to Search activity icon

### PR DESCRIPTION
**Pull request by kmjennison**

### CODE REVIEW

File: src/utils/__tests__/adHelpers.test.js [hunk 1]
```diff
@@ -81,10 +81,7 @@ describe('Tab Tracking Methods', () => {
   })
 
   it('returns zero tabs today when the last tab day count is not set in localStorage', () => {
-    localStorageMgr.getItem.mockReturnValueOnce(
-      STORAGE_TABS_LAST_TAB_OPENED_DATE,
-      moment(mockNow).utc().toISOString()
-    )
+    localStorageMgr.getItem.mockReturnValueOnce(undefined)
```
Comment (jedtan) [inline src/utils/__tests__/adHelpers.test.js:84]: Is this related?

**Review by jedtan — APPROVED** [review]

File: src/utils/__tests__/adHelpers.test.js (hunk 1 above)
Comment (kmjennison) [reply src/utils/__tests__/adHelpers.test.js:84]: @jedtan No, but great catch, haha. This was an error that was causing noise in test output that annoyed me.
